### PR TITLE
Share price chart

### PR DESCRIPTION
**Pull request by iherger**

### CODE REVIEW

File: src/components/Charts/PriceChart/PriceChart.tsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  margin-bottom: ${(props) => props.theme.spaceUnits.m};
+  margin-left: 0;
+`;
+
+export const PriceChart: React.FC<PriceChartProps> = (props) => {
+  const theme = useTheme();
+  const fund = useFund();
+
+  const showSecondaryData =
+    props.queryType == 'depth' && (props.depth === '1d' || props.depth === '1w' || props.depth === '1m') ? true : false;
```
Comment (iherger) [inline src/components/Charts/PriceChart/PriceChart.tsx:33]: I think we should remove the off-chain data for the `1m` default view.

File: src/theme.ts [hunk 2]
```diff
@@ -152,6 +152,43 @@ export const light = {
...
+    },
+    crosshair: {
+      line: {
+        stroke: 'rgb(29,29,29)', // primaryDark
+        strokeWidth: 1,
+        strokeOpacity: 0.75,
+        strokeDasharray: '6 6',
+      },
+    },
+  },
```
Comment (iherger) [inline src/theme.ts:191]: Is this code still necessary?

File: src/theme.ts [hunk 3]
```diff
@@ -200,6 +237,44 @@ export const dark = {
...
+    },
+    crosshair: {
+      line: {
+        stroke: '#e5dfdf', // primaryDark
+        strokeWidth: 1,
+        strokeOpacity: 0.75,
+        strokeDasharray: '6 6',
+      },
+    },
+  },
```
Comment (iherger) [inline src/theme.ts:276]: Also here, is this necessary?

File: src/components/Routes/Fund/FundOverview/FundOverview.tsx [hunk 4]
```diff
@@ -36,4 +42,8 @@ export const FundOverview: React.FC<FundOverviewProps> = ({ address }) => (
   </Grid>
 );
 
+// pass it the query object and a function that it can apply to the query object
+// post processing function prop, nivo calls refetch,
+// uselazyquery
+
```
Comment (iherger) [inline src/components/Routes/Fund/FundOverview/FundOverview.tsx:48]: Remove comments.

File: src/hooks/useQuery.ts [hunk 5]
```diff
@@ -74,3 +74,11 @@ export const useTheGraphQuery = <TData = any, TVariables = OperationVariables>(
   const client = useTheGraphClient();
   return useContextQuery<TData, TVariables>(client, query, options);
 };
+
+export const useLazyTheGraphQuery = <TData = any, TVariables = OperationVariables>(
```
[the remark this answers is not in the document]
Comment (erin-koen) [reply src/hooks/useQuery.ts:78]: yeah this was a remnant of our pre-metrics service days when we were querying shareprice data directly from the subgraph. I've removed it.

File: src/components/Routes/Fund/FundPerformanceChart/FundPerformanceChart.tsx [hunk 6]
```diff
@@ -0,0 +1,183 @@
...
+  const parsedOffchainDataByDepth = React.useMemo(() => {
+    return (
+      offchainDataByDepth &&
+      ([{ id: 'off-chain', name: 'Interim share price movements', type: 'line', data: offchainDataByDepth }] as Serie[])
+    );
+  }, [offchainDataByDepth]);
+
+  const parsedOnchainDataByDate = React.useMemo(() => {
+    return (onchainDataByDate
+      ? [{ id: 'on-chain', name: 'On-chain share price', type: 'area', data: onchainDataByDate }]
```
[the remark this answers is not in the document]
Comment (erin-koen) [reply src/components/Routes/Fund/FundPerformanceChart/FundPerformanceChart.tsx:151]: the id key here is in an object that's passed to the chart as data. The two instances where `on-chain` is used are never passed  simultaneously

[the remark this answers is not in the document]
Comment (erin-koen) [reply src/components/Routes/Fund/FundPerformanceChart/FundPerformanceChart.tsx:150]: Haven't changed this but open to if you think it's a problem @fubhy

**Pull request merged**